lblod/frontend-form-builder
# SFB-71: fix codelist options showing in preview

## ID
 [SFB-71](https://binnenland.atlassian.net/browse/SFB-71)

 ## Description

When adding a concept scheme to a field that is able to show options, these options should be visible. The information was not parsed into the meta graph of the store before which makes that showing the options was not possible. Because of PR https://github.com/lblod/app-form-builder/pull/7 this is possible.

Got all the information from the store and added it to it's own `concept-schemes.ttl` file. This file is than parsed in the preview store meta graph. Now the selectors are able to show the corresponding options.

 ## Type of change


 ## How to test

Get a concept scheme configuration from the codelijsten tab. Copy the json and go back to the builder tab. In the builder tab create a field with a type that is able to show concept scheme options. for example the radio buttons or just the selector. Past the json configuration into the textarea field and see if all the options are visible in the preview form. 

 ## Links to other PR's

 - https://github.com/lblod/app-form-builder/pull/7
 - https://github.com/lblod/frontend-form-builder/pull/66
